### PR TITLE
Add Theme Provider

### PR DESCRIPTION
**Pull request by joshgachnang**

### CODE REVIEW

File: packages/ui/src/Page.tsx [hunk 1]
```diff
@@ -99,6 +99,7 @@ export class Page extends React.Component<PageProps, {}> {
           width="100%"
         >
           {this.renderHeader()}
+          {/* TODO: Use Theme */}
           {this.props.loading === true && (
             <Spinner color={Unifier.theme.darkGray as any} size="md" />
```
Comment (abbiecoghlan) [inline packages/ui/src/Page.tsx:104]: @joshgachnang any reason this one is a to-do rather than using it?

File: packages/ui/src/Heading.tsx [hunk 2]
```diff
@@ -1,53 +1,53 @@
...
-    );
-  }
+  };
+
+  const lines = 0;
+  // if (numberOfLines) {
+  //   lines = numberOfLines;
+  // } else if (inline) {
+  //   lines = 1;
+  // }
```
Comment (abbiecoghlan) [inline packages/ui/src/Heading.tsx:47]: Do we need this in here still?

**Review by abbiecoghlan — APPROVED** [review]
One TODO question + we should wait to merge this as discussed, but otherwise LGTM!

File: packages/ui/src/Heading.tsx (hunk 2 above)
Comment (joshgachnang) [reply packages/ui/src/Heading.tsx:47]: Nope! Also made this a functional component.

**Pull request merged**